src/Util/typeUtils.ts

```diff
@@ -0,0 +1,29 @@
+import OlBaseLayer from '@hanreev/types-ol/ol/layer/Base';
+import OlLayerGroup from '@hanreev/types-ol/ol/layer/Group';
+import OlLayer from 'ol/layer/Layer';
+import OlImageWMS from 'ol/source/ImageWMS';
+import OlTileWMS from 'ol/source/TileWMS';
+import ImageLayer from 'ol/layer/Image';
+import TileLayer from 'ol/layer/Tile';
+
+export type ArrayTwoOrMore<T> = [T, T] & T[];
+
+export function isArrayTwoOrMore<T>(value: T[]): value is ArrayTwoOrMore<T> {
+  return value?.length > 1;
+}
```

Comment:
`ol-util` is not written in typescript so this type guards don't make sense there

Reply:
`ArrayTwoOrMore` is a type that is used at several points in `react-geo`. I included this guard here because it is nice to have in a project that uses react-geo.

Reply:
The `base-util` is actually written in typescript.

Reply:
i was getting confused with all the util libs :laughing:

Reply:
I opened https://github.com/terrestris/base-util/pull/381